ui/src/components/data-entities/customTooltip.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+export default class CustomTooltip extends Component {
+
+  getReactContainerClasses() {
+    return ['custom-tooltip'];
+  }
+
+  stringToHTML = (str) => {
+    let parser = new DOMParser();
+    let doc = parser.parseFromString(str, 'text/html');
+    return doc.body;
+  };
+
+  render() {
+
+    let data;
+
+    if (typeof this.props.value === 'object') {
+
+      data = JSON.stringify(this.props.value)?.replaceAll(/["\{\}]/g,'').trim();
+
+      if (data.length > 0) {
+        return (<div className={"custom-tooltip"} >
+              <span>
+                <div className={"body"}>{data.split(",").map(item => <p>{item}</p>)}</div>
+              </span>
+            </div>
+        );
+      }
+    }
+
+    return (<div className={"custom-tooltip ag-tooltip-hiding"} ></div>)
+
+
+  }
+}
```

Comment:
I m not sure we need to add a custom tooltip. ag-grid knows how to deal with large text: https://blog.ag-grid.com/wrapping-column-header-text/

Reply:
I dont want to effect heights here